src/index.jsx

```diff
@@ -3,28 +3,66 @@ import React from 'react';
 import { render } from 'react-dom';
 import { AppContainer } from 'react-hot-loader';
 import './index.css';
-import GenericErrorBoundary from './components/genericErrorBoundary';
 import Routes from './routes';
 import registerTooltip from './utils/registerTooltip';
 
 require('typeface-roboto');
 
+let handleError = (error, info) => {
+  // eslint-disable-next-line no-console
+  console.error(error, info);
+};
+
 if (process.env.NODE_ENV === 'production') {
   Raven.config(
     'https://77916a47017347528d25824beb0a077e@sentry.io/1225660'
   ).install();
+  handleError = Raven.captureException;
 }
 
 // handle sticky tooltip for all charts
 registerTooltip();
 
+class GlobalErrorBoundary extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  componentDidCatch(error, info) {
+    this.setState({ error });
+    handleError(error, info);
+  }
+
+  render() {
+    if (this.state.error) {
+      return (
+        <p style={{ textAlign: 'center', fontSize: '1.5em' }}>
+          <span>
+            There has been a critical error. We have reported it. If the issue
+            is not fixed within few hours please file an issue:
+            <br />
+            <a
+              href="https://github.com/mozilla/firefox-health-dashboard/issues/new"
+              target="_blank"
+              rel="noopener noreferrer">
+              {'https://github.com/mozilla/firefox-health-dashboard/issues/new'}
+            </a>
+          </span>
+        </p>
+      );
+    }
+
+    return this.props.children;
+  }
+}
 const root = document.getElementById('root');
 const load = () =>
   render(
     <AppContainer>
-      <GenericErrorBoundary critical>
+      <GlobalErrorBoundary>
```

Comment:
I think the only difference with `withErrorBoundary` is that it does not show the error message (e.g. `<ErrorMessage error={error}>{super.render()}</ErrorMessage>`) but a customized UI.

I wonder if we can do something like this in routes.jsx:
```
const ComponentWithMessage = (
  <div>File a precious issue!</div>
);

export default withErrorBoundary(Routes, ComponentWithMessage);
```

And then make this change to the signature:
```
const withErrorBoundary = (WrappedComponent, ShowOnError = null) => {
```

This is not that important but I thought we might be able having two different classes to handle error boundaries.